tests/integration/misbehaving_extension/misbehaving.cpp

```diff
@@ -57,37 +59,34 @@ PyObject*
 call_fn(PyObject*, PyObject* args)
 {
     PyObject* callback;
-    if (!PyArg_ParseTuple(args,"O", &callback))
-    {
+    if (!PyArg_ParseTuple(args, "O", &callback)) {
         PyErr_SetString(PyExc_ValueError, "Failed to parse arguments");
         Py_RETURN_NONE;
     }
-    Py_BEGIN_ALLOW_THREADS
-    start_threads(callback);
+    Py_BEGIN_ALLOW_THREADS start_threads(callback);
     join_threads();
-    Py_END_ALLOW_THREADS
-    Py_RETURN_NONE;
+    Py_END_ALLOW_THREADS Py_RETURN_NONE;
```

Comment:
Ooh, this isn't great. `clang-format` is making a mess of `Py_BEGIN_ALLOW_THREADS` and `Py_END_ALLOW_THREADS`.

For now, maybe we should tell `clang-format` to ignore `tests/integration/*` ...